editor/src/clj/editor/properties_view.clj

```diff
@@ -82,16 +82,25 @@
                                                      (ui/run-later (.selectAll t))))
                                                  (ui/user-data! t ::selection-at-focus nil)))))
 
-(defmulti customize! (fn [control update-fn] (class control)))
+(defmulti customize! (fn [control _ _] (class control)))
 
-(defmethod customize! TextField [^TextField t update-fn]
+(defmethod customize! TextField [^TextField t update-fn cancel-fn]
   (doto t
     (GridPane/setHgrow Priority/ALWAYS)
     (ui/on-action! update-fn)
     (ui/auto-commit! update-fn)
-    (select-all-on-click!)))
-
-(defmethod customize! TextArea [^TextArea t update-fn]
+    (select-all-on-click!)
+    (ui/bind-key! "Esc" (fn []
+                          (cancel-fn t)
+                          (when-let [parent (.getParent t)]
+                            (.requestFocus parent))))
+    (ui/bind-key! "Enter" (fn []
+                            (update-fn t)
+                            (if (zero? (.getLength (.getSelection t)))
+                              (.selectAll t)
+                              (.deselect t))))))
```

Comment:
Update(commit) input and select value:

https://github.com/defold/defold/assets/2209596/30e049fb-bf55-4e92-9cd9-15d818595d22